packages/aztec.js/src/note/index.js

```diff
@@ -207,19 +237,29 @@ note.utils = noteUtils;
  * @method fromValue
  * @param {string} publicKey hex-string formatted recipient public key
  * @param {number} value value of the note
- * @param {string} owner owner of the not if different from the public key
+ * @param {string} noteOwner owner of the note if different from the public key
+ * @param {Object} access mapping between Ethereum addresses being granted view access of a note
```

Comment:
Not an object. Or at least not strictly an object.

Reply:
Have updated - `access` is now just an `Array`